train.py

```diff
@@ -101,15 +116,13 @@ def run_training_loop(args: argparse.Namespace):
     #########################################################
     ###################### TRAINING LOOP ####################
     #########################################################
-
-
     day = time.strftime(f'%Y-%m-%d-%H%M')
-    exp_name = f'{args.experiment}_{args.model_name_or_path}_{day}'
+    exp_name = f'{args.experiment}_{args.model_name}_{day}'
     # WandB init and config (based on argument dictionaries in imports/globals cell)
     wandb.init(
         name=exp_name,
-        project='rf-bert',
-        entity='jscuds',
+        project=os.environ.get('WANDB_PROJECT', 'rf-bert'),
+        entity=os.environ.get('WANDB_ENTITY', 'jscuds'),
```

Comment:
This pulls the environment variable, and if it's not set to anything defaults to `rf-bert` and `jscuds`...correct?

Reply:
yeah exactly. this is so I can type `WANDB_ENTITY='jxmorris12' python train.py` because otherwise it'll try to connect me to your account and break